File: 백준/Bronze/1546. 평균/평균.js
const fs = require("fs");
const path = require("path");
const input = fs
  .readFileSync(
    process.platform === "linux"
      ? "/dev/stdin"
      : path.join(__dirname, "/input.txt"),
  )
  .toString()
  .trim()
  .split("\n");

const solution = (input) => {
  const N = Number(input[0]);
  const arr = input[1].split(" ").map(Number);

  const max = Math.max(...arr);
  const avg =
    arr.map((v) => (v / max) * 100).reduce((acc, cur) => acc + cur, 0) / N;
  console.log(avg);
};

solution(input);

File: 백준/Bronze/1546. 평균/평균.cc
#include <iostream>

using namespace std;

int main()
{
    int N;
    cin >> N;
    
    double arr[N];
    int max=0;
    for(int i=0;i<N;i++){
        cin >> arr[i];
        if(arr[i]>max) max=arr[i];
    }

    double sum=0;
    for(int i=0;i<N;i++){
        sum+=(arr[i]/max)*100;
    }

    cout << sum/N << endl;

}